caikit/core/data_model/base.py

```diff
@@ -70,29 +76,43 @@ def __new__(mcs, name, bases, attrs):
         protobufs, it can be named in the tuple class variable _private_slots and will
         automatically be added to __slots__.
         """
-        # Get all fields in protobufs with same name as class,
-        # except for DataBase, which has no matching protobufs
-        fields = ()
 
-        # Protobufs fields can be divided into these categories, which are used to automatically
```

Comment:
I rearranged things a bit so that it was less confusing to keep track of the pre-defined empty variables and the names they were assigned to in `attrs`